tests/integration/test_target_snowflake.py

```diff
@@ -271,15 +260,15 @@ def test_loading_unicode_characters(self):
         self.assertEqual(
             table_unicode,
```

Comment:
There is an existing bug here. table_unicode select is: `SELECT * FROM {}.test_table_unicode`

Snowflake doesn't guarantee the column order and this test sometimes fails sometimes passes. Can we change it to `SELECT * FROM {}.test_table_unicode ORDER BY c_int` so it will be always ordered as the expected rows.

Reply:
Will take care of that.